spec/site/tlc/clock_spec.rb

```diff
@@ -49,6 +49,7 @@
       Validator::Site.connected do |task,supervisor,site|
         prepare task, site
         with_clock_set CLOCK do
+          sleep 10
```

Comment:
sleep() will block everything else, including responding to watchdog messages, timers, etc. instead use task.sleep()

Reply:
Fixed. Thanks